debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -236,16 +237,21 @@
                                                      COLLECTION_BLACKLIST,
                                                      AUTO_DISCOVER_MEMBERS,
                                                      DATABASE_WHITELIST,
-                                                     DATABASE_BLACKLIST);
+                                                     DATABASE_BLACKLIST,
+                                                     CommonConnectorConfig.TOMBSTONES_ON_DELETE);
 
     protected static Field.Set EXPOSED_FIELDS = ALL_FIELDS;
 
+    public MongoDbConnectorConfig(Configuration config) {
+        super(config);
+    }
+
     protected static ConfigDef configDef() {
         ConfigDef config = new ConfigDef();
         Field.group(config, "MongoDB", HOSTS, USER, PASSWORD, LOGICAL_NAME, CONNECT_BACKOFF_INITIAL_DELAY_MS,
                     CONNECT_BACKOFF_MAX_DELAY_MS, MAX_FAILED_CONNECTIONS, AUTO_DISCOVER_MEMBERS,
                     SSL_ENABLED, SSL_ALLOW_INVALID_HOSTNAMES);
-        Field.group(config, "Events", DATABASE_WHITELIST, DATABASE_BLACKLIST, COLLECTION_WHITELIST, COLLECTION_BLACKLIST);
+        Field.group(config, "Events", DATABASE_WHITELIST, DATABASE_BLACKLIST, COLLECTION_WHITELIST, COLLECTION_BLACKLIST, CommonConnectorConfig.TOMBSTONES_ON_DELETE);
```

Comment:
Maybe the `CommonConnectorConfig` should have method(s) that will add all common fields.

Reply:
I thought about it, but `group()` just accepts a var-args, so adding anything returned by another method will be cumbersome to use. So I've left that for another time.

Reply:
My idea was something like

```java
class CommonConnectorConfig
public void fieldGroup(ConfigDef config, String groupName, Field... fields) {
// add common fields that belong to group named groupName to the fields array
   Field.group(config, groupName, fields);
}

// and invoke it via
CommonConnectorConfig.fieldGroup(config, "Events", DATABASE_WHITELIST, DATABASE_BLACKLIST, COLLECTION_WHITELIST, COLLECTION_BLACKLIST, CommonConnectorConfig.TOMBSTONES_ON_DELETE);
```

Reply:
Can we revisit this one separately? For now I'd prefer to leave control of the options to expose in the individual connector config types. I.e. it might be that there's an option declared in common which is only exposed by two of the three connectors. I think this needs some more consideration.